demos/security-trifecta/src/pages/login/LoginCard.tsx

```diff
@@ -6,17 +6,70 @@ import {
   IonCardSubtitle,
   IonCardTitle,
   IonIcon,
+  IonSelect,
+  IonSelectOption,
 } from '@ionic/react';
 import { logInOutline } from 'ionicons/icons';
+import { useEffect, useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { login } from '../../utils/auth';
+import { UnlockMode } from '../../models/UnlockMode';
+import { getUnlockMode, setUnlockMode, canUseLocking } from '../../utils/session-vault';
+import { Device } from '@ionic-enterprise/identity-vault';
+
+interface UnlockOptions {
```

Comment:
Small change, but I'd prefer this to be called `UnlockOption` since it'll be used as a single entry within an array of possible options (a variable that is an array of these items would use the plural, but not the type definition of the item itself).

Reply:
Done